src/Template/MailPreview/index.ctp

```diff
@@ -47,18 +47,18 @@
         {
             public function welcome($user)
             {
-                return $this // Returning the chain is a good idea :)
+                return $this // ' . __d('debug_kit', 'Returning the chain is a good idea') . ' :)
```

Comment:
These are comments, there is no code here.

Reply:
I know.... But this comments are showed in UI

Reply:
They are? But then they will not show translated with this here either, as this is not valid PHP syntax.

Reply:
Yes, it does, because this sample code is inside a PHP '$code' variable. That is, it is not HTML but a PHP example inside a variable. So I concatenated the function I18N with the variable string content